### PR TITLE
Update docs for verify and chart attributes in helm_release

### PR DESCRIPTION
**Pull request by jrhouston**
### Description

This PR adds some small updates to clear up some confusion in the documentation for `helm_release`

Related to #537 

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### CODE REVIEW

File: website/docs/r/release.html.markdown [hunk 1]
```diff
@@ -48,18 +48,29 @@ resource "helm_release" "example" {
...
-resource "helm_release" "local" {
+resource "helm_release" "example" {
   name       = "my-local-chart"
   chart      = "./charts/example"
 }
 ```
 
+## Example Usage - Chart URL
+
+In case a Chart is not available from a repository, a path may be used:
```
Comment (dak1n1) [inline website/docs/r/release.html.markdown]: Nice! This is really helpful to know.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -26,7 +26,9 @@ labels: bug
 ```
 
 ### Debug Output
-<!--Please provider a link to a GitHub Gist containing the complete debug output: https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the debug output in the issue; just paste a link to the Gist.-->
+<!--Please provider a link to a GitHub Gist containing the complete debug output: https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the debug output in the issue; just paste a link to the Gist. 
+
+NOTE: In addition to Terraform debugging, please set HELM_DEBUG=1 to enable debugging info from helm.-->
```
Comment (dak1n1) [inline .github/ISSUE_TEMPLATE/bug-report.md:31]: TIL :smile:

File: website/docs/r/release.html.markdown [hunk 3]
```diff
@@ -69,7 +80,7 @@ The following arguments are supported:
 * `devel` - (Optional) Use chart development versions, too. Equivalent to version '>0.0.0-0'. If version is set, this is ignored.
 * `version` - (Optional) Specify the exact chart version to install. If this is not specified, the latest version is installed.
 * `namespace` - (Optional) The namespace to install the release into. Defaults to `default`
-* `verify` - (Optional) Verify the package before installing it. Defaults to `false`
+* `verify` - (Optional) Verify the package before installing it. Helm uses a provenance file to verify the integrity of the chart, this must be hosted alongside the chart. For more information see the [Helm Documentation](https://helm.sh/docs/topics/provenance/). Defaults to `false`
```
Comment (dak1n1) [inline website/docs/r/release.html.markdown]: Grammar suggestion: a semi-colon links together two independent clauses that are closely related in thought.

```suggestion
* `verify` - (Optional) Verify the package before installing it. Helm uses a provenance file to verify the integrity of the chart; this must be hosted alongside the chart. For more information see the [Helm Documentation](https://helm.sh/docs/topics/provenance/). Defaults to `false`.
```

File: website/docs/r/release.html.markdown [hunk 4]
```diff
@@ -69,7 +80,7 @@ The following arguments are supported:
 * `devel` - (Optional) Use chart development versions, too. Equivalent to version '>0.0.0-0'. If version is set, this is ignored.
 * `version` - (Optional) Specify the exact chart version to install. If this is not specified, the latest version is installed.
 * `namespace` - (Optional) The namespace to install the release into. Defaults to `default`
```
Comment (dak1n1) [inline website/docs/r/release.html.markdown]: Unrelated, missing period.

```suggestion
* `namespace` - (Optional) The namespace to install the release into. Defaults to `default`.
```

**Review by dak1n1 — COMMENTED** [review]
These are some great changes! I learned some things from it. It's very clear and easy to understand. I left a couple minor grammar syntax suggestions.

**Comment by jrhouston** [comment]
Thanks for the suggestions @dak1n1!

**Pull request merged**